translations/default/ModulesGAnalyticsAdmin.xlf

```diff
@@ -102,6 +102,36 @@
         <target>Settings for cancelled order states updated successfully</target>
         <note>Line: 246</note>
       </trans-unit>
+      <trans-unit id="0824d52b9e53469ffa87f750985865c9">
+        <source>Enable Google Analytics 4</source>
+        <target>Enable Google Analytics 4</target>
+        <note>Line: 100</note>
+      </trans-unit>
+      <trans-unit id="233b2b197d921eebd8c0f45e4354fea7">
+        <source>Canceled order states</source>
+        <target>Canceled order states</target>
+        <note>Line: 170</note>
+      </trans-unit>
+      <trans-unit id="899811bf0d6c66e1435a14005939c792">
+        <source>Choose order states, in which you consider the given order canceled. This will usually be the default "Canceled" state, but some shops may have extra states like "Returned" etc.</source>
```

Comment:
```suggestion
        <source>Choose order states, in which you consider the given order canceled. This will usually be the default "Canceled" state, but some stores may have extra states like "Returned", etc.</source>
```